src/OutgoingMessage.ts

```diff
@@ -57,16 +59,24 @@ export default class OutgoingMessage extends NativeOutgoingMessage {
     }));
     // eslint-disable-next-line no-invalid-this
     this.done();
+
+    // eslint-disable-next-line no-invalid-this
+    return this;
```

Comment:
why returning this?

Reply:
because the method signature we are overriding is changed.